weave-js/codegen.ts

```diff
@@ -0,0 +1,18 @@
+import {CodegenConfig} from '@graphql-codegen/cli';
+
+const config: CodegenConfig = {
+  overwrite: true,
+  schema: './wb_schema.gql',
```

Comment:
This is totally fine for now! Thank you Connie (: We can add some automation around this in the future. Since weave is open source, we copy a simplified schema here with just the min required edges